.gitignore

```diff
@@ -8,3 +8,9 @@ include/igloo
 .coveralls.yml
 doxygen/
 *.ps.gz
+*.zip
+*.json
+/include/blitz
+/include/boost/
+/include/metis.h
+/include/suitesparse/
```

Comment:
Nice, this should shield us against accidentally checking in third party headers that could get out of sync.

On my linux setup, i just have these headers installed somewhere like '/usr/include/' where I tell the makefile to look. On windows, things are a bit different I guess.